etc/settings.cfg.sh

```diff
@@ -25,9 +25,7 @@ export EUPSPKG_SOURCE=${EUPSPKG_SOURCE:-git}
 export EUPS_PKGROOT=$LSSTSW/distserver/production
 
 # the location of source repositories
-BASE='git://git.lsstcorp.org/LSST'
-SBASE='https://stash.lsstcorp.org/scm'
-export REPOSITORY_PATTERN="$BASE/sims/%(product)s.git|$BASE/DMS/%(product)s.git|$BASE/DMS/devenv/%(product)s.git|$BASE/DMS/testdata/%(product)s.git|$BASE/external/%(product)s.git|$SBASE/sim/%(product)s.git"
+export REPOFILE=${LSSTSW}/etc/repos.yaml
```

Comment:
I don't like this. "We'll only have a couple of potential locations" is speculation, and duplicate names would actually be undetected under the current regime. Having two mechanisms for something that is only relevant to our build system is a completely unnecessary increase in maintainance workflow. I suggest we talk about this further in the repos reorg session on Tuesday.